open-metadata-implementation/access-services/asset-lineage/asset-lineage-spring/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/server/spring/AssetLineageResource.java

```diff
@@ -37,4 +37,21 @@ public GUIDListResponse publishEntities(@PathVariable String serverName,
                                             @PathVariable String entityType) {
         return restAPI.publishEntities(serverName, userId, entityType);
     }
+
+    /**
+     * Find the entity by guid and publish the context for it
+     *
+     * @param serverName name of server instance to call
+     * @param userId     the name of the calling user
+     * @param guid       the guid of the entity to build context
+     * @param entityType the name of the relationship type
+     * @return a list of unique identifiers (guids) of the available entity with the given type provided as a response
+     */
+    @GetMapping(path = "/publish-entity/{entityType}/guid/{guid}")
```

Comment:
I am not sure if we have agreed on Egeria-wide REST API endpoint design practices but having /guid/ in the path seems bit odd to me. I would prefer to have it without this part and call parameter id instead. The fact that is guid is just a choice and format of an identifier. 

So my proposal for this endpoint is: /publish-entity/{entityType}/{id}

@popa-raluca any thoughts from the previous practice with rest endpoints?